File: DataStructure/6차_과제/7-2/py_src/7_2.py
from header7_2 import treeNode
from func7_2_1 import *

def main():
    F11=treeMake(120, None, None)
    F10=treeMake(55, None, None)
    F9=treeMake(100, None, None)
    F8=treeMake(200, None, None)
    F7=treeMake(68, F10, F11)
    F6=treeMake(40, None, None)
    F5=treeMake(15, None, None)
    F4=treeMake(2, F8, F9)
    F3=treeMake(10, F6, F7)
    F2=treeMake(0, F4, F5)
    F1=treeMake(0, F2, F3)

    postorder(F1)

    print(f"C:\\의 용량{F2.data}")
    print(f"D:\\의 용량{F3.data}")
    print(f"내 컴퓨터 전체 용량{F1.data}")

if __name__=="__main__":
    main()

File: DataStructure/6차_과제/7-2/c_src/7-2.c
#include <stdio.h>
#include "7-2.h"

int main() {
    treeNode* F11 = fileData(120, NULL, NULL);
    treeNode* F10 = fileData(55, NULL, NULL);
    treeNode* F9 = fileData(100, NULL, NULL);
    treeNode* F8 = fileData(200, NULL, NULL);
    treeNode* F7 = fileData(68, F10, F11);
    treeNode* F6 = fileData(40, NULL, NULL);
    treeNode* F5 = fileData(15, NULL, NULL);
    treeNode* F4 = fileData(2, F8, F9);
    treeNode* F3 = fileData(10, F6, F7);
    treeNode* F2 = fileData(0, F4, F5);
    treeNode* F1 = fileData(0, F2, F3);

    Datasum(F1);

    printf("C:\\의 용량: %d\n", F2->data);
    printf("D:\\의 용량: %d\n", F3->data);
    printf("내 컴퓨터의 전체 용량: %d\n", F1->data);

    return 0;
}